### ISSUE TITLE
{Linked:Bug12953710; Linked:Bug15457051;Linked:Bug5002719}[Adaptive Cards - Inputs with Validation] - Screen Reader is conveying incorrect position for the options present under what color do you want? combo box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9022]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
Turn on screen reader using caps + spacebar key.

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Press tab key and navigate to the 'Inputs with Validation' link and press enter key to activate it.
4. Press the tab key and navigate to the What color do you want? compact combo box.
5. Press enter key to activate it.
6. Navigate the list and observe the issue.

**Actual Result:**
Screen Reader is conveying incorrect position for the menu items present under What color do you want? (Compact). There are 3 items are present but while navigating on it screen reader is conveying information as **'Red 1 of 4 selected current.'**

**Expected Result:**
Screen Reader should convey the correct position for the menu items present under What color do you want? (Compact). For the first option it should announce as **'Red 1 of 3 selected.'**

**Note:**
Same issue is repro throughout the application for the same control type.
Same issue is repro with NVDA.

**User Impact:**
It will impact the screen reader user as they will get confused and assume there are 4 items are present on the screen.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Screen Reader is conveying incorrect position for the options present under what color do you want combo box](https://github.com/user-attachments/assets/29f2f29a-4519-498a-956d-e9aa0339c8aa)

https://github.com/user-attachments/assets/29b8ec62-62d0-4036-b147-2de1db1f6075

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Tracking bug id -  https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/12953710

**Comment by vagpt** [comment]
Bug https://dev.azure.com/MicrosoftIT/OneITVSO/_workitems/edit/15457051
